modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/plugins/field-sidebar/components/FormsFieldSidebar.es.js

```diff
@@ -105,7 +105,9 @@ export const FormsFieldSidebar = ({title}) => {
 			displaySettings={Object.keys(focusedField).length > 0}
 			editingLanguageId={editingLanguageId}
 			fieldTypes={sortFieldTypes(
-				fieldTypes.filter(({group}) => group === 'basic')
+				fieldTypes.filter(
+					({group}) => group === 'basic' || group === 'interface'
+				)
```

Comment:
From what I understand in the old Sidebar, he put it together in `Basic Elements` belonging to the group `basic` and `interface` and added any other group in `Customized Elements` which is the custom fields that customers can add, to support this, the best thing here would be to remove the filter and just pass `fieldTypes`.

cc @natocesarrego what do you think?